tests/integration/conftest.py

```diff
@@ -64,13 +64,7 @@ def start_trino(image_tag=None):
 
 
 def wait_for_trino_workers(host, port, timeout=180):
-    request = TrinoRequest(
-        host=host,
-        port=port,
-        client_session=ClientSession(
-            user="test_fixture"
-        )
-    )
+    request = TrinoRequest(host=host, port=port, client_session=ClientSession(user="test_fixture"))
```

Comment:
arguably less readable than before.

Reply:
This seems linked to the setting of the line length. It seems black wants to shove as much on the same line if permitted by the line length. No other knobs available. black is fairly opinionated.

Reply:
@mdesmet if you add a trailing comma after `client_session=ClientSession(user="test_fixture"`, Black will format the code is a similar way to how it was formatted previously.